### PR TITLE
MM-36880 Load other teams you can join when deleting cache

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Display the option for other teams you can join after deleting cache & data.

#### Ticket Link
Fixes: #5512
JIRA: https://mattermost.atlassian.net/browse/MM-36880

#### Release Note
```release-note
Fixed a bug where the option to join another team was not present after deleting cache & data
```

### CODE REVIEW

File: app/init/global_event_handler.js [hunk 1]
```diff
@@ -213,6 +214,7 @@ class GlobalEventHandler {
 
         await dispatch(loadConfigAndLicense());
         await dispatch(loadMe(user));
+        dispatch(getTeams());
```
Comment (larkox) [inline app/init/global_event_handler.js:217]: We are awaiting the rest of the dispatches in this function. Any reason not to wait for this one?

**Review by larkox — APPROVED** [review]
LGTM, just a small question.

File: app/init/global_event_handler.js (hunk 1 above)
Comment (enahum) [reply app/init/global_event_handler.js:217]: The other two are sort of required when the rest of the UI loads, the last one is just something that we really don't depend on

**Comment by enahum** [comment]
@josephbaylon circleCI is having a partial outage that is affecting building the app

**Comment by enahum** [comment]
@josephbaylon the app should be in the QA Build channel

**Review by josephbaylon — APPROVED** [review]
LGTM. `Open teams you can join` option is still displayed and is tappable even when cache is deleted.

**Pull request merged**